source/features/github-bugs.css

```
@@ -102,6 +102,29 @@ svg.octicon-package + a.Truncate[href*='/releases/download/'] > .Truncate-text {
 	white-space: wrap;
 }
 
+/* Fix notification layout */
+/* Info: https://github.com/refined-github/refined-github/pull/8208 */
+/* Test: https://github.com/notifications */
+.gap-3:has([action='/notifications/beta/update_view_preference']) {
+	gap: 0 !important; /* Turns .gap-3 off */
+}
+
+:has([action='/notifications/beta/update_view_preference'])
+	.gap-4.pl-3:has(div > .js-socket-channel.js-updatable-content) {
+	padding-left: 0 !important; /* Moves .pl-3 to child element that toggles visibility */
+}
+
+:has([action='/notifications/beta/update_view_preference'])
+	.gap-4.pl-3
+	> div
+	> .js-socket-channel.js-updatable-content
+	> a.h6 {
+	padding-left: var(
+		--base-size-24,
+		24px
+	) !important; /* Turns .pl-3 into .pl-4 */
+}
```

```suggestion
/* Matches .d-md-block */
@media (min-width: 768px) {
	/* Pulls the alert container to the right when empty */
	.gap-4.pl-3:has(> .d-md-block > [data-url^="/notifications/beta/recent_notifications_alert"]):not(:has(a.h6)) {
		margin-right: calc(0px - var(--base-size-16, 16px) * 2);
	}
}
```


While not as "clean" it's much easier to reason about.

---

You know what you need better than I do—be my guest 🤷🏻‍♂️